core/state_transition.go

```diff
@@ -650,6 +660,8 @@ func (st *stateTransition) innerExecute() (*ExecutionResult, error) {
 				}
 				st.state.AddBalance(params.OptimismL1FeeRecipient, amtU256, tracing.BalanceIncreaseRewardTransactionFee)
 			}
+			operatorFeeCost := st.evm.Context.OperatorCostFunc(new(big.Int).SetUint64(st.gasUsed()), st.evm.Context.Time)
+			st.state.AddBalance(params.OptimismOperatorFeeRecipient, operatorFeeCost, tracing.BalanceIncreaseRewardTransactionFee)
```

Comment:
We shouldn't access extra state unnecessarily. Even if the cost is zero, this change should only be applied after isthmus activates. Otherwise, this breaks users that use `debug_executionWitness` to cache previous state witnesses to determine how to prove a block transition.
This is also likely the reason the op-program-compat workflow [fails](https://app.circleci.com/pipelines/github/ethereum-optimism/optimism/80991/workflows/6cfe622b-cdb0-4016-b760-136044f82ecd/jobs/3248722) in the monorepo.